tests/erlang_tests/trap_exit_flag.erl

```diff
@@ -31,12 +31,16 @@ start() ->
     0.
 
 test_nocatch() ->
+    MyPid = self(),
     Pid = spawn_opt(fun proc/0, []),
     erlang:link(Pid),
+    erlang:link(Pid),
+    {links,[Pid]} = erlang:process_info(MyPid,links),
```

Comment:
Thank you. You probably need to run erlfmt here.

Reply:
Ahh, thanks. :blush:

Reply:
It also seems that the CI tests create some extra link that I do not get when I run locally causing the test to fail:
`{links,[<0.0.0>,<0.76.0>]}} `. I expect `{links,[<0.76.0>]}}` in the test.

Reply:
Right, there is a difference when running the tests with BEAM as opposed to AtomVM. Indeed, we run tests with both VMs to align expectations, and BEAM does link with <0.0.0> (a process that doesn't exist with AtomVM).

You can run the test locally with BEAM using -b flag:
```shell
./tests/test-erlang -b trap_exit_flag
```

or all tests
```shell
./tests/test-erlang -b
```

Reply:
Seems it works without the -b flag too?
`./build/tests/test-erlang  trap_exit_flag`

Anyway I updated the test case to only take into account the relevant Pid.

Reply:
Ok, I noticed now that -b is against the BEAM! Thanks.